src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -76,14 +64,23 @@ function renderValue(v: Value) {
   }
 }
 
-function buildCellRenderer(column: AnonymizedResultColumn, bucketColumns: BucketColumn[]) {
+function renderBucketValue(column: AnonymizedResultColumn, bucketColumns: BucketColumn[]) {
   const bucketColumn = bucketColumns.find((c) => c.name === column.name);
-  return (v: Value) => {
+  return (v: Value, row: TableRowData) => {
     // see note on `ellipsis` above
     if (v === null) {
       return nullCell();
     } else {
-      return valueCell(bucketColumn, v);
+      if (
+        bucketColumn &&
+        (bucketColumn.type === 'integer' || bucketColumn.type === 'real') &&
+        bucketColumn.generalization &&
+        !row.isStarBucketRow
```

Comment:
If we want the better tooltip we need this to be a case in itself.

Reply:
Maybe `groupingLabelRenderer` is more appropriate.

Reply:
> Maybe groupingLabelRenderer is a more appropriate name.

hm, as mentioned above, I'd favor simplicity and consistency of the entire context over a strictly accurate name. Let me know how this works for you.

Reply:
Sorry, posted in the wrong thread from phone. As for the first comment, we need to return `<Tooltip title="Suppress bin">*</Tooltip>` in a separate case, otherwise we return `<Tooltip title="*>*</Tooltip>

Reply:
re first comment:

It works fine, as it is yet again back to the previous behavior (which I find marginally nicer, so I kept it), as contrasted with the one I mentioned in the PR description.

Currently the *-bucket row renders as before the PR: 
![Screenshot from 2022-01-04 09-53-07](https://user-images.githubusercontent.com/5735525/148033556-f07e04a3-a51e-4988-8860-11f28023a842.png)

Sorry about the confusion

Reply:
Ah, I thought the new behavior was desired.